deck.md

```diff
@@ -594,47 +844,162 @@ http://localhost:3000/login
 
 class: branded
 
-# Step 4: Decorators, Hooks, Autoload
+# Step 8: Config
 
-- in the previous step we generated a JWT token that can be used to access protected routes. In this step we're going to create a protected route and allow access only to authenticated users via a Fastify decorator and a Fastify hook
+- It is preferable to use a config file (.env) to keep track of environment variables used in the system, for example, the JWT secret we put directly into the code in the previous step.
 
-- [`fastify-autoload`](https://github.com/fastify/fastify-autoload) is a convenience plugin for Fastify that loads all plugins found in a directory and automatically configures routes matching the folder structure
+- This makes it easier to deploy the same code into different environments as we only need to change the config file rather than having to deploy different copies of the code.
+
+- Typically config would not be committed to a repository and would be managed in different environments accordingly. For example, locally, developers might want to see everything from the log, whereas in Production we wouldn't want to generate huge log files and might restrict the log, outputting just errors.
```

Comment:
```suggestion
- Typically config values are not committed to a repository and they are managed with environment variables. An example would be the logging level: in production it's usually better to have only errors, while in a dev environment it may be useful to show more.
```